### DISCUSSION TITLE
recordsize of a child filesystem to (80k) smaller than a parent (128k)

### DISCUSSION THREAD

**Discussion by MikeJakubik** [openzfs/zfs#12956] [Q&A]
Hello,

I set the recordsize of a child filesystem to (80k) to a smaller than a parent (128k) 

NAME                         USED  AVAIL     REFER  MOUNTPOINT
zroot                       3.11G  1.68T       96K  /zroot
zroot/ROOT                   953M  1.68T       96K  none
zroot/ROOT/default           953M  1.68T      953M  /
zroot/jails                  504K  1.68T      120K  /usr/local/jails
zroot/jails/staging-app-01    96K  1.68T       96K  /usr/local/jails/staging-app-01
zroot/jails/staging-app-02    96K  1.68T       96K  /usr/local/jails/staging-app-02
zroot/jails/staging-db-01     96K  1.68T       96K  /usr/local/jails/staging-db-01
zroot/jails/staging-db-02     96K  1.68T       96K  /usr/local/jails/staging-db-02

Does this make any sense since the root FS is larger?

NAME         PROPERTY    VALUE    SOURCE
zroot/jails  recordsize  128K     default
zroot/jails/staging-app-01  recordsize  128K     default
zroot/jails/staging-db-01  recordsize  8K       local

Thanks.

**Comment by amotin** [comment]
Children and parent data set record sizes are not related in any way.  You can set any power of 2 you like between 512 bytes (I really don't recommend to go too low) and allowed maximum anywhere.

**Comment by MikeJakubik** [comment]
I set it to 8k per PostgreSQL recommendation for high IOPS, but if it will write at 128k then I guess it's pointless??
